CMakeLists.txt

```diff
@@ -615,8 +608,20 @@ else()
   set(swift_dir swift_static)
 endif()
 
+# TODO(compnerd) honour lib vs lib64
```

Comment:
I don't think that the movement was necessary.  The reason for the organization was that `library_kind` and `swift_dir` are not used by the installation of the module/doc files.

Reply:
I moved it below the `set(swift_dir..` as I am also getting the static lib install working and wanted `${swift_dir}` taken into account.